src/types/account.ts

```diff
@@ -82,6 +83,7 @@ export enum AccountSearchType {
   ContractStorage,
   ContractCode,
   InternalTxReceipt,
+  // SecureAccount,
```

Comment:
same as explorer.. why added as comment?

Reply:
same as explorer, the commented out SecureAccount type was added to the AccountSearchType enum in account.ts as a placeholder for future secure account search functionality. Removed it in collector also in the latest commit